network/nordvpn/doinst.sh

```diff
@@ -1,3 +1,14 @@
+# Patched by Peter Christy <pete.christy.uk@gmail.com>
+
+# update configuration and shared library cache for linker to find .so files
+echo "/usr/lib64/nordvpn" > /etc/ld.so.conf.d/nordvpn.conf
```

Comment:
The info file has a download for both 32 and 64bit. But this is hardcoded to lib64?